### ISSUE TITLE
Dependency clarification

### ISSUE THREAD

**Issue by eric-haibin-lin** [dmlc/gluon-nlp#1025]
I've seen a few cases where gluonnlp 0.8 is used with a very old mxnet version on sagemaker and some functionalities won't work. Shall we auto-detect the mxnet version and print a warning if it does not meet the minimum required version?

**Comment by szha** [comment]
Agreed that this is a gap. Though we didn't just specify mxnet as a dependency because of the different variants of mxnet in pip, we should still check for the version.
